### PR TITLE
Fix image version comment

### PR DESCRIPTION
**Pull request by nervo**
**What this PR does / why we need it**: Fix a typo in a comment

### CODE REVIEW

File: stable/phpmyadmin/values.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Bitnami WordPress image version
+## Bitnami PhpMyAdmin image version
```
Comment (carrodher) [inline stable/phpmyadmin/values.yaml]: Can you use `phpMyAdmin` instead? To be consistent with https://github.com/kubernetes/charts/pull/5752

**Comment by carrodher** [comment]
Thanks for the contribution, before continue, you need to [sign the CLA](https://git.k8s.io/community/CLA.md#the-contributor-license-agreement), also you should increment the chart version in the `Chart.yml` according to [semver](http://semver.org/):

> Given a version number `MAJOR.MINOR.PATCH`, increment the:
> - MAJOR version when you make incompatible API changes,
> - MINOR version when you add functionality in a backwards-compatible manner, and
> - PATCH version when you make backwards-compatible bug fixes.
> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

**Comment by nervo** [comment]
Everything (CLA, phpMyAdmin and chart version) should be ok :)

**Comment by carrodher** [comment]
The current version in https://github.com/kubernetes/charts/blob/master/stable/phpmyadmin/Chart.yaml is `version: 0.1.2` so you need to update it to `version: 0.1.3`. The rest is ok to me and ready for merge

**Pull request closed without merging**